Orange/widgets/visualize/owmosaic.py

```diff
@@ -512,6 +512,18 @@ def update_graph(self):
         spacing = self.SPACING
         bar_width = self.BAR_WIDTH
 
+        def get_counts(attr_vals, values):
+            if attr_vals == "":
```

Comment:
Nice. Could you also add a docstring to this function describing what it does and how it handles zero counts?